### ISSUE TITLE
RegionOpeningException thrown during online alter

### ISSUE THREAD

**Issue by junegunn** [OpenTSDB/asynchbase#117]
Hi, we noticed that operations can fail with RegionOpeningException when online alter is ongoing.

- HBase 0.98.6
- Asynchbase 1.6.0

```
org.hbase.async.RemoteException: org.apache.hadoop.hbase.exceptions.RegionOpeningException: Region contacts,9eb851b0,1436233402166.bb1cf7ed55f23ccf46eddac13b0d3059. is opening on **REDACTED**,60020,1432199329355
	at org.apache.hadoop.hbase.regionserver.HRegionServer.getRegionByEncodedName(HRegionServer.java:2758)
	at org.apache.hadoop.hbase.regionserver.HRegionServer.getRegion(HRegionServer.java:4256)
	at org.apache.hadoop.hbase.regionserver.HRegionServer.get(HRegionServer.java:2894)
	at org.apache.hadoop.hbase.protobuf.generated.ClientProtos$ClientService$2.callBlockingMethod(ClientProtos.java:29921)
	at org.apache.hadoop.hbase.ipc.RpcServer.call(RpcServer.java:2031)
	at org.apache.hadoop.hbase.ipc.CallRunner.run(CallRunner.java:108)
	at org.apache.hadoop.hbase.ipc.RpcExecutor.consumerLoop(RpcExecutor.java:116)
	at org.apache.hadoop.hbase.ipc.RpcExecutor$1.run(RpcExecutor.java:96)
	at java.lang.Thread.run(Thread.java:745)
...
```

With the default Java API, online alter normally does not incur error on the client-side thanks to the automatic retrials, but it seems that Asynchbase is not aware of `RegionOpeningException` and does not retry on it but only on `NotServingRegionException` and `RegionMovedException`.

**Comment by manolama** [comment]
Ah thanks, that should be a fairly easy fix.

**Comment by manolama** [comment]
Patched in 41df65924e1e3a74edbeb0749c065db81e5ec11c

**Comment by junegunn** [comment]
Thanks!
Any plan to release 1.7.0?
